dll/win32/shell32/wine/shellpath.c

```diff
@@ -3099,6 +3099,32 @@ BOOL WINAPI SHGetSpecialFolderPathW (
                             szPath) == S_OK;
 }
 
+#ifdef __REACTOS__
+HRESULT WINAPI SHGetFolderLocationHelper(HWND hwnd, int nFolder, REFCLSID clsid, LPITEMIDLIST *ppidl)
+{
+    HRESULT hr;
+    IShellFolder *psf;
+    LPITEMIDLIST parent, child;
+    EXTERN_C HRESULT SHBindToObject(IShellFolder *psf, LPCITEMIDLIST pidl, REFIID riid, void **ppvObj);
```

Comment:
It's better to declare function in appreciate place.

Reply:
What is the proper place? It was **you** that placed the declaration for this function in precomp.h